app/Helpers/render/comment.php

```diff
@@ -113,7 +114,7 @@ function RenderArticleComment(
     $deleteIcon = '';
 
     if ($user && $user === $localUser || $allowDelete) {
-        $img = "<img src='" . asset('assets/images/icon/cross.png') . "' width='16' height='16' alt='delete comment'/>";
+        $img = Blade::render('<x-fas-xmark class="text-red-700 h-5 w-5" alt="delete comment"/>');
```

Comment:
Apologies. It looks like Lighthouse is still complaining:

![Screenshot 2024-01-08 at 5 35 56 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/f0492ca8-973c-4a36-bd16-24c8e5701592)

It seems like it'll only be happy if we remove the `alt` from the icon itself and instead use a `title` attribute on the `<a />` tag.

Note, I believe this a11y check has been failing this entire time. I'm only raising this because IMO it's critical to clean up while we're giving these pieces of the UX some love.